src/IncrementalInference.jl

```diff
@@ -534,6 +534,11 @@ include("TetherUtils.jl")
 include("CliqStateMachine.jl")
 include("CliqStateMachineUtils.jl")
 
+#EXPERIMENTAL parametric
+include("ParametricSolveTree.jl")
+include("ParametricCliqStateMachine.jl")
+include("ParametricUtils.jl")
```

Comment:
nice that its in three separate files, as we figure out how to consolidate and dispatch we can reduce the amount of code in these files down to a bear minimum through a few release cycles